org.gnome.Radio.json

```diff
@@ -0,0 +1,105 @@
+{
+  "app-id" : "org.gnome.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "finish-args" : [
+     "--socket=x11",
+     "--share=ipc"
+  ],
+    "modules": [
+	{
+	    "name": "intltool",
+	    "cleanup": [ "*" ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+		    "sha256": "67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
+		}
+	    ]
+	},
+	{
+            "name": "cogl",
+            "config-opts": [
+		"--disable-cogl-gst",
+		"--disable-gtk-doc",
+		"--enable-xlib-egl-platform",
+		"--enable-wayland-egl-platform"
+            ],
+            "cleanup": [
+		"/share/gtk-doc",
+		"/share/cogl/examples-data"
+            ],
+            "sources": [
+		{
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/cogl/1.22/cogl-1.22.8.tar.xz",
+                    "sha256": "a805b2b019184710ff53d0496f9f0ce6dcca420c141a0f4f6fcc02131581d759"
+		}
+            ]
+	},  
+	{
+            "name": "clutter",
+            "config-opts": [
+		"--disable-gtk-doc",
+		"--enable-egl-backend",
+		"--enable-wayland-backend"
+            ],
+            "cleanup": [
+		"/share/gtk-doc"
+            ],
+            "sources": [
+		{
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/clutter/1.26/clutter-1.26.4.tar.xz",
+                    "sha256": "8b48fac159843f556d0a6be3dbfc6b083fc6d9c58a20a49a6b4919ab4263c4e6"
+		}
+            ]
+	},
+	{
+	    "name": "clutter-gtk",
+	    "cleanup": [
+		"/share/gtk-doc"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://download.gnome.org/sources/clutter-gtk/1.8/clutter-gtk-1.8.4.tar.xz",
+		    "sha256": "521493ec038973c77edcb8bc5eac23eed41645117894aaee7300b2487cb42b06"
+		}
+	    ]
+	},
+	{
+	    "name" : "champlain-gtk-0.12",
+	    "sources" : [
+		{
+		    "type" : "archive",
+		    "url" : "https://download.gnome.org/sources/libchamplain/0.12/libchamplain-0.12.16.tar.xz",
+		    "sha256" : "4a7e31cf7889669aebf04631543af64435edd989685159b804911c6005db908d"
+		}
+	    ]
+	},
+	{
+	    "name" : "geocode-glib-1.0",
+	    "sources" : [
+		{
+		    "type" : "archive",
+		    "url" : "https://ftp.gnome.org/pub/GNOME/sources/geocode-glib/3.24/geocode-glib-3.24.0.tar.xz",
+		    "sha256" : "19c1fef4fd89eb4bfe6decca45ac45a2eca9bb7933be560ce6c172194840c35e"
+		}
+	    ]
+	},
+	{
+	    "name" : "gnome-radio",
+	    "sources" : [
+		{
+		    "type" : "archive",
+		    "url" : "download.gnome.org/sources/gnome-radio/64/gnome-radio-64.0.tar.xz",
```

Comment:
this URL is invalid. (it's not a proper URL)

Reply:
Try https://download.gnome.org/sources/gnome-radio/64/gnome-radio-64.0.tar.xz
Fixed in the commit.

Reply:
Fixed.

The URL is https://download.gnome.org/sources/gnome-radio/64/gnome-radio-64.0.tar.xz and I also fixed the sha256sum.